src/Uno.UWP/System/Launcher.macOS.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using AppKit;
+using Windows.Foundation;
+using Windows.Storage;
+
+namespace Windows.System
+{
+	public static partial class Launcher
+	{
+		public static IAsyncOperation<bool> LaunchFolderPathAsync(string path)
+		{
+			if (path is null)
+			{
+				throw new ArgumentNullException(nameof(path));
+			}
+
+			var success = NSWorkspace.SharedWorkspace.OpenFile(path);
+			return Task.FromResult(success).AsAsyncOperation();
```

Comment:
I'm curious, does the timing here match Windows?

Reply:
Yes, I think I tested that - UWP does not wait for the folder to close or such.